### PR TITLE
Make check interval configurable for the runner

### PR DESCRIPTION
**Pull request by PirminTapken**
## Description

Make check interval configurable for the runner in order to reduce load on gitlab API

## Migrations required

NO

### CODE REVIEW

**Comment by PirminTapken** [comment]
One checkmark can be added to https://github.com/npalm/terraform-aws-gitlab-runner/issues/393 by this.

File: variables.tf [hunk 1]
```diff
@@ -314,6 +314,12 @@ variable "runners_request_spot_instance" {
   default     = true
 }
 
+variable "runners_check_interval" {
+  description = "defines the interval length, in seconds, between new jobs check."
+  type        = number
+  default     = 0
```
Comment (npalm) [inline variables.tf:320]: Ok by me, other option could be to set the default to 3 https://docs.gitlab.com/runner/configuration/advanced-configuration.html

**Review by npalm — APPROVED** [review]
@PirminTapken thanks LGTM

**Comment by npalm** [comment]
@PirminTapken please can you check the CI, looks like there is a format error.

File: main.tf [hunk 2]
```diff
@@ -133,14 +133,15 @@ locals {
       runners_root_size                 = var.runners_root_size
       runners_iam_instance_profile_name = var.runners_iam_instance_profile_name
       runners_use_private_address_only  = var.runners_use_private_address
-      runners_use_private_address       = !var.runners_use_private_address
+      runners_use_private_address       = ! var.runners_use_private_address
```
Comment (npalm) [inline main.tf:136]:
```suggestion
      runners_use_private_address       = !var.runners_use_private_address
```

**Review by npalm — CHANGES_REQUESTED** [review]
Fix formatting

**Comment by npalm** [comment]
@PirminTapken did you had time to check the review comment?

**Comment by vgropp** [comment]
@npalm i stepped in to fix formatting and default to 3 seconds as @PirminTapken seems quite busy

**Review by npalm — APPROVED** [review]

**Pull request merged**